tests/ArraySchemaValidatorTest.php

```diff
@@ -36,28 +36,26 @@ public function testArrayOfStringsValid(): void {
   }
 
   public function testArrayOfStringsLegacyArrays(): void {
-    $validator = new ArraySchemaValidator(darray['array_of_strings' => varray['test', 'list', 'of', 'strings']]);
+    $validator = new ArraySchemaValidator(darray['array_of_strings' => vec['test', 'list', 'of', 'strings']]);
     $validator->validate();
 
     expect($validator->isValid())->toBeTrue();
     $validated = $validator->getValidatedInput();
     expect(C\count($validated['array_of_strings'] ?? vec[]))->toBeSame(4);
   }
 
-  public function testArrayOfStringsLegacyAndHackArrays(): void {
```

Comment:
removed `testArrayOfStringsLegacyAndHackArrays` since it ended up being the same as `testArrayOfStringsValid` if we changed the `varray` -> `vec`